cmd/empire/factories.go

```diff
@@ -227,3 +229,21 @@ func newHBReporter(key, env string) (reporter.Reporter, error) {
 	// since it thinks MultiReporter is a struct literal.
 	return append(reporter.MultiReporter{}, empire.DefaultReporter, r), nil
 }
+
+// Auth provider =======================
+
+func newAuthProvider(c *cli.Context) (dockerauth.AuthProvider, error) {
+	provider := dockerauth.NewMultiAuthProvider()
+	provider.AddProvider(dockerauth.NewEcrAuthProvider(ecr.New(newConfigProvider(c))))
+
+	dockerConfigFile, err := os.Open(c.String(FlagDockerAuth))
+	if err == nil {
```

Comment:
Same here, if a docker auth file was provided but can't be opened, we should bubble up that error.